manifests/0000_30_cluster_csr_approver_01_serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: cluster-csr-approver-controller
```

Comment:
what is this, is this used anywhere?

Reply:
oh I see, the client builder in CPC uses the `SAControllerClientBuilder` that allows using different SAs per controller. TIL.

Reply:
Resolving this comment as I moved the definitions along with @deads2k's suggestion.